charts/cortx/values.yaml

```
@@ -61,6 +61,11 @@ kafka:
   # -- Overridden min.insync.replicas config for the transaction topic
   transactionStateLogMinIsr: 2
 
+  initContainers:
+  - name: zookeeper-watcher
```

```suggestion
  - name: wait-for-zookeeper
```

Nitpick, but I prefer the "wait-for" naming that Bitnami uses, it does what is says. "watcher" makes me think of some kind of continuous monitor.